packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/CryptoPayment.tsx

```diff
@@ -388,25 +392,32 @@ const CryptoPayment = ({ planPrice }: ICryptoPayment) => {
           component="p"
           variant="body1"
           className={classes.error}
+          data-cy="label-crypto-payment-error"
         >
           <Trans>Failed to create a charge</Trans>
         </Typography>
       }
       {cryptoPayment && pendingCryptoInvoice &&
         <>
           <div className={classes.rowBox}>
-            <Typography>Total</Typography>
+            <Typography data-cy="label-total-crypto-title">
+              Total
+            </Typography>
             <div className={classes.pushRightBox}>
-              <Typography>
+              <Typography data-cy="label-total-crypto-price">
                 {pendingCryptoInvoice.currency?.toUpperCase()} {pendingCryptoInvoice.amount}
               </Typography>
             </div>
           </div>
           <Divider />
           {!selectedCurrency && currencies &&
             <>
-              <Typography>Select a cryptocurrency</Typography>
-              <div className={classes.availableCurrencies}>
+              <Typography data-cy="label-select-cryptocurrency">
+                Select a cryptocurrency
```

Comment:
Has nothing to do with this PR, but since I see it...
```suggestion
                <Trans>Select a cryptocurrency</Trans>
```